bin/cli.js

```diff
@@ -9,13 +9,17 @@ const cli = meow(
     $ multi-semantic-release
 
   Options
-    --sequential-init  Avoid hypothetical concurrent initialization collisions.
+    --dry-run Dry run mode.
     --debug Output debugging information.
+    --sequential-init  Avoid hypothetical concurrent initialization collisions.
     --first-parent Apply commit filtering to current branch only.
+    --deps.bump Define deps version updating rule. Allowed: override, satisfy, inherit.
+    --deps.release Define release type for dependent package if any of its deps changes. Supported values: patch, minor, major, inherit.
```

Comment:
I think you may want to change these to be with dashes instead of dots, as I'm not sure on the cross platform portability of using dots in arguments.

Reply:
I'm planning to introduce **msr** prefix to avoid clashes with **semrel** own flags. So this part will be refactored a bit later.